docs/preview/03-Features/sinks/azure-application-insights.md

```diff
@@ -174,35 +174,44 @@ ILogger logger = loggerConfig.CreateLogger();
 
 ### Q: Where can I initialize the logger in an ASP.NET Core application or other hosted service?
 
-Simply use the `UseSerilog` extension method on `IHostBuilder` which accepts an `Action<HostBuilderContext, IServiceProvider, LoggerConfiguration>`.
-This Action gives you access to the configuration and the configured services.
-
 If the connection string is stored as a secret in -for instance- Azure KeyVault, the `ISecretProvider` from [Arcus secret store](https://security.arcus-azure.net/features/secret-store) can be used to retrieve the connection string.
 
+Use the `UseSerilog` extension method on `IHostBuilder` which accepts an `ILogger` and use the Serilog's static `Log.Logger` property to setup the logger. This makes sure that we can reload the logger afterwards (`CreateBootstrapLogger`). This is needed because otherwise an endless loop will occur when the Arcus secret store is used (secret store required logging, logging requires secret store).
```

Comment:
Building a `IHost` instance without Serilog will still provide you with a `ILogger`, so that's fine, using the `bootstrap logger` is only needed if you want to store the Application Insights connection string in the secret store.

Reply:
So it is not mandatory.  However, while reading it, I thought it was mandatory.
As I read it, it looks like we must do that.  So, maybe we should rephrase this a bit ?